Assets/Scripts/Scenarios/LaunchMainMenuScenario.cs

```diff
@@ -25,11 +25,11 @@ public void Construct(IUIRoot uiRoot, LoadingService loadingService, LevelLoadin
 
         protected void Start()
         {
-            //var view = _uiRoot.Open<StartSplashScreen>();
-            //_uiRoot.Open<MainMenu>();
-            //view.Closed += OnViewClosed;
-            var view = _uiRoot.Open<StartSplashScreen>();
-            view.Closed += OnSplashScreenClosed;
+            var mainMenuView = _uiRoot.Open<MainMenu>();
+            mainMenuView.Closed += OnMainMenuClosed;
```

Comment:
Subscibe to thi method not neccesary. Pls remove.

Reply:
Removed